### PR TITLE
Bulk edit UI bugs

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] [ch22874]

- Add hover states to checkboxes
- Add sidebar animation to modal sidebar
- Change "Due In" to "Due Date"

### CODE REVIEW

File: src/js/views/globals/modal/modal_views.js [hunk 1]
```diff
@@ -73,6 +75,13 @@ const ModalView = View.extend({
   },
 });
 
+const SidebarModalView = ModalView.extend({
```
Comment (paulfalgout) [inline src/js/views/globals/modal/modal_views.js:78]: since we're making a view here let's move over the 
```js
      className: 'modal--sidebar',
      bodyClass: 'sidebar',	
```
in fact.. we should probably just go ahead a make a tall and a small and move the classes out of that app entirely.

File: src/js/views/patients/worklist/worklist-list.scss [hunk 2]
```diff
@@ -14,6 +14,14 @@
   .fa-check-square {
     color: color(green);
   }
+
+  .fa-square {
+    color: $black-90;
```
Comment (paulfalgout) [inline src/js/views/patients/worklist/worklist-list.scss]: 90 or 80?  90 below is disabled

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/patients/worklist/worklist-list.scss (hunk 2 above)
Comment (LisaManresa) [reply src/js/views/patients/worklist/worklist-list.scss]: Ah thanks. Forgot we had two instances. What do you think about making these styles more global. We could add them to buttons.scss and create a `button-checkbox` class.

File: src/js/views/patients/worklist/worklist-list.scss [hunk 3]
```diff
@@ -7,12 +7,13 @@
...
-  cursor: default;
   padding: 16px 0;
   width: 16px;
+}
 
-  .fa-check-square {
-    color: color(green);
+.work-list__item:hover {
+  .fa-square {
+    color: $black-20;
```
Comment (paulfalgout) [inline src/js/views/patients/worklist/worklist-list.scss:16]: why is this needed and how is it work-list specific?

File: src/sass/modules/buttons.scss [hunk 4]
```diff
@@ -204,3 +204,26 @@
...
+  .fa-square {
+    color: $black-80;
+
+    &:hover {
+      color: $black-20;
+    }
+  }
+
+  &:disabled {
+    cursor: default;
```
Comment (paulfalgout) [inline src/sass/modules/buttons.scss:217]: is cursor even needed?

File: src/sass/modules/buttons.scss [hunk 5]
```diff
@@ -204,3 +204,26 @@
     color: $black-80;
   }
 }
+
+.button--checkbox {
```
Comment (paulfalgout) [inline src/sass/modules/buttons.scss:208]: would it work to do:
```css
.button--checkbox {
  color: $black-80;
  &:hover...
  &:disabled...
 .fa-check-square...
}
```
without .fa-square?  I can't remember if the icon has to have the color on it entirely.. even if it does maybe `.fa-square` could be color inherit?

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/patients/worklist/worklist-list.scss (hunk 3 above)
Comment (LisaManresa) [reply src/js/views/patients/worklist/worklist-list.scss:16]: Mark asked that the checkbox have the hover state when a user hovers anywhere on the row.

File: src/sass/modules/buttons.scss (hunk 4 above)
Comment (LisaManresa) [reply src/sass/modules/buttons.scss:217]: Yeah, cursor is needed because our reset scss sets the cursor for all buttons to pointer.

File: src/js/views/patients/worklist/worklist-list.scss (hunk 3 above)
Comment (paulfalgout) [reply src/js/views/patients/worklist/worklist-list.scss:16]: 🙄

**Review by paulfalgout — APPROVED** [review]
for whenever this branch shows the latest update
